tests/AbstractApiClientTest.php

```diff
@@ -0,0 +1,56 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of the Bushido\ApiClient package.
+ *
+ * (c) Wojciech Nowicki <wnowicki@me.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace BushidoTests\ApiClient;
+
+use Bushido\ApiClient\ApiClient;
+use Bushido\ApiClient\Exceptions\BadResponseException;
+use Bushido\ApiClient\Exceptions\TransportException;
+use Bushido\ApiClient\Exceptions\WrongResponseException;
+use PHPUnit\Framework\TestCase;
+
+class AbstractApiClientTest extends TestCase
```

Comment:
It may worth making a note that these tests will fail if there is no internet connection available, or if http://httpbin.org is not reachable.

Reply:
Added to README.md